tron/core/job_scheduler.py

```diff
@@ -85,6 +85,26 @@ def schedule(self):
             return
         self.create_and_schedule_runs()
 
+    def update_from_job_scheduler(self, job_scheduler):
+        """ Update a job scheduler by copying another. """
+        old_job = self.get_job()
+        new_job = job_scheduler.get_job()
+
+        old_job.update_from_job(new_job)
+
+        # Since job upadting only copies equality attributes (defined in the Job
+        # class), we need to now enable or disable the job depending on the
+        # new job says so.
+        if old_job.enabled is not new_job.enabled and \
```

Comment:
Do you need this? I think you could simplify 98-107 to:
```
if old_job.config_enabled is not new_job.config_enabled:
  old_job.enabled = new_job.config_enabled
old_job.config_enabled = new_job.config_enabled
```

Reply:
I do want to keep thing as they are for 2 reasons:
1. Logging: Although reconfiguration is logged elsewhere, I'd like re-enabling/disabling by tronctl to be logged as well. And of course, those logs are sort of unnecessary if the job was already enabled/disabled to begin with, hence the conditions.
2. Enable()/disable(): Although setting job.enabled is enough technically, I use enable()/disable() so that tronfig is like tronctl when it comes to modifying job state. Additionally, those functions do things like schedule runs as well as cancel pending ones.